src/provable/field-bigint.ts

```diff
@@ -1,40 +1,50 @@
-import { HashInput, ProvableExtended } from './provable.js';
+import { Binable } from './binable.js';
```

Comment:
this file is integrated into snarkyjs by this PR and should be reviewed in its entirety!